content/en/docs/concepts/signals/traces.md

```diff
@@ -229,6 +230,10 @@ Sample Span:
 }
 ```
 
+As is implied by the present of a parent span ID, spans can be nested as
+child spans that represent sub-operations. This allows them to accurately
+model work done in an application.
+
```

Comment:
```suggestion
Spans can be nested, as is implied by the presence of a parent span ID:
child spans represent sub-operations. This allows spans to more accurately
capture the work done in an application.
```

Reply:
Hmm, I'm not quite happy with the last sentence (both the original and the revised one).
Could we just drop it? If not, I'll try to think of something better -- something about capturing operation details or about fine-grainedness?

Reply:
I'd prefer not to drop it, since it's a crucial description of why they can be nested. Maybe it needs to be expanded into something more descriptive. Since programs are made up of operations, and some of which are nested (A calls B), we need a way to represent that work in diagnostics. The ability for spans to be nested is what accomplishes that.

Reply:
Ok, then let's keep the last sentence. Are you ok with the edits @cartermp?

Reply:
Yep! For some reaosn github isn't letting me accept the suggestion, though, even after doing what it says 🙃 

<img width="351" alt="image" src="https://user-images.githubusercontent.com/6309070/182877198-3baa4b8a-71b1-4e67-a806-d6c9c8b0f279.png">

Reply:
Maybe you're logged in using a different ID?